docs/setup-steps-1.4.0.md

```diff
@@ -0,0 +1,265 @@
+# 1.4.0
+
+## Pre-requisites
+
+CTK should be deployed with the required dockers.
+
+* compliance-toolkit-service: 1.4.0
+* compliance-toolkit-ui: 1.4.0
+* compliance-toolkit-batch-job: 1.4.0
+
+To deploy Compliance Toolkit, we require the below mandatory services:
```

Comment:
To successfully deploy Compliance Toolkit, below mentioned services are mandatorily required.